### DISCUSSION TITLE
How to mask akhq own consumer group in ConsumerGroup column?

### DISCUSSION THREAD

**Discussion by juguyard** [tchiotludo/akhq#714] [Q&A] [answered]
Hi,

Almost all in the title.
I'm installing a new akhq instance for my confluent cloud cluster. So far so good, but I decided to give this new akhq limited ACL, only whats required. And I specified a group.id in configuration. On my onprem cluster, akhq had no right limitation, it was global cluster ACL, and I didn't specify any group.id.

Now, on my new akhq instance I see akhq own consumer group in ConsumerGroup column for each topic. Not a blocker, but it would be great if I could mask it, because this information is not interesting. Any idea?

**Comment by tchiotludo** [comment, marked as answer]
Really strange ! 
If I understand, you are setting the configuration : 
```
akhq:
  clients-defaults:
    consumer:
      properties:
        group.id: Akhq
```
And now you see this consumer group on all the topic you are consuming from akhq ? right ? 
If it's true, it shouldn't, since AKHQ must not commit offset as I know.

**Reply by ghost** [reply]
is it possible that this behavior is related to the fact that the following parameter is added in the config :  enable.auto.commit: true

**Reply by tchiotludo** [reply]
clearly, you should never add this configuration to AKHQ

**Reply by ghost** [reply]
Indeed, after deleting this configuration, the akhq group is no longer displayed in the console. 
Thanks a lot !

**Reply by juguyard** [reply]
+1, merci Ludovic
